Hajohns/catalog templates

haileyajohnson opened the pull request:
- allow user-supplied custom template fragments
- restyle catalog and dataset access pages

lesserwhirls commented:
@haileyajohnson - to test locally, try:

` ./gradlew.bat :it:freshInstallTest`

lesserwhirls commented:
Can you do the following:

1) In `gradle\any\dependencies.gradle`, bump the version of chronicle map to `net.openhft:chronicle-map:3.15.1` (currently 3.14.5)

   https://github.com/Unidata/thredds/blob/5.0.0/gradle/any/dependencies.gradle#L208

2) Also in dependencies.gradle, remove lines 212-218

   https://github.com/Unidata/thredds/blob/5.0.0/gradle/any/dependencies.gradle#L212-L218

haileyajohnson commented:
Cool, I can access the ncss pages now and fixed the stuff I broke in those templates (and rebased on the right branch and now CI seems to be passing again).

lesserwhirls commented on tds/src/main/webapp/WEB-INF/altContent/startup/threddsConfig.xml:
```diff
@@ -48,10 +48,13 @@
-    <catalogCssUrl>tdsCat.css</catalogCssUrl>
-    <datasetCssUrl>tdsDst.css</datasetCssUrl>
-    <openDapCssUrl>tdsDap.css</openDapCssUrl>
+    LOOK
```
?

haileyajohnson replied:
Meant to remove both of these...

lesserwhirls replied:
Cool :-)

cwardgar commented on tds/src/main/webapp/WEB-INF/templates/pointFragments.html:
```diff
@@ -16,6 +23,7 @@
           west:  /*[[${#numbers.formatDecimal(boundingBox?.lonMin, 0, 3)}]]*/ "-90.0000",
           east:  /*[[${#numbers.formatDecimal(boundingBox?.lonMax, 0, 3)}]]*/ "90.0000"
       };
+      };
```
Extra closing parenthesis here, FYI.

lesserwhirls commented:
Thanks @cwardgar!
